packages/frontend/src/Build.spec.ts

```diff
@@ -943,3 +943,38 @@ test('purposely fail prereqs, expect an error to show going to build', async ()
     expect(errorMessage).toBeDefined();
   });
 });
+
+
+// The below mocks are the same across all tests.
+const setupPlatformMocks = (isMac: boolean, isWindows: boolean, isLinux: boolean) => {
```

Comment:
Suggestion: `const setupPlatformMocks = (platform: string) => {`
and
`vi.mocked(bootcClient.isMac).mockResolvedValue(platform == 'macOS'); ...`
and
`{ platform: 'macOS', shouldShow: true }, ...`
seems simpler?